spec/request/swagger_spec.rb

```diff
@@ -610,6 +613,25 @@
       end
     end
 
+    describe 'bb' do
+      include BB::ClientHelpers
```

Comment:
In an effort to keep things DRY, could we double check and see if behavior doesn't already exist in another helper? If we need to rename it to be more service agnostic, maybe MHV::ClientHelpers or something.